dev/Manifest.toml

```diff
@@ -0,0 +1,150 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
I'm not familiar with Artifacts, is this `Manifest` intended to be committed?

Reply:
This `Manifest.toml` is committed as when executing this script I want it to use the currently checked out version of TimeZones.jl. You can specify a relative path like that in a `Manifest.toml` file but not in the `Project.toml`. This is same reason we have a manifest in the docs/ dir.

Reply:
This is specific for the `dev/generate_artifacts.jl` code but not specific to artifacts themselves

Reply:
Ah gotcha, yeah I figured that it was similar to `docs`, but wasn't sure! Thanks for clearing that up.